pom.xml

```diff
@@ -48,9 +48,14 @@
 	<dependency>
       <groupId>io.jenkins</groupId>
       <artifactId>configuration-as-code</artifactId>
-      <version>1.0</version>
+      <version>1.2</version>
       <scope>test</scope>
 	</dependency>
+	<dependency>
+		<groupId>org.eclipse.jdt</groupId>
+		<artifactId>org.eclipse.jdt.annotation</artifactId>
```

Comment:
The spotbugs ones would be better imho. Can we all just standardise on the spotbugs ones (and bonus they are the same package as the original findbugs ones)

Reply:
I agree 🙌 Plus I started https://github.com/jenkinsci/plugin-pom/pull/128

Reply:
Any thoughts on where we should go with this now?  Looks like the mentioned PR is hitting some roadblocks.

Reply:
We can switch to spotbugs in this repo 😅 and later remove it once plugin pom is ready...